pages/test_analytics/importing_junit_xml.md.erb

```diff
@@ -4,6 +4,15 @@ While most test frameworks have a built-in JUnit XML export feature, these JUnit
 
 {:toc}
 
+## Mandatory JUnit XML Attributes
+
+The following attributes are mandatory for the `<testcase>` element:
+
+* `classname` - Full class name for the class the test method is in.
+* `name` - Name of the test method.
+
+[Read more about the JUnit XML file format.](https://llg.cubic.org/docs/junit/)
```

Comment:
@olyism , is it okay for us to link to a third-party website?

Reply:
Hi @gchan yes, it's ok to link to a third-party website

Reply:
+1 No issues with linking to a third-party source. It can be a great help for doc maintenance. 😊

Is Linux Lighting Group an authority on the topic? I tried to look for an official spec, but I couldn't see anything.

Also, rather than linking the full sentence, we should have the link on the part relevant to the page. It's a good practice to format the sentence with context before the link, too. In this case, something like:

```suggestion
To learn more about the JUnit XML file format, see [JUnit XML reporting file format](https://llg.cubic.org/docs/junit/) from Linux Lighting Group.
```

Note, I wouldn't normally add the "from Linux Lighting Group" bit, but I see there's no visual difference between an internal and external link.

Reply:
I can't find a better authority on the XML file format.

* https://www.ibm.com/docs/en/developer-for-zos/14.1?topic=formats-junit-xml-format seems less comprehensive to Linux Lightning Group. Itself references a source that is not available anymore.
* JUnit's doc doesn't contain a schema or reference either :( https://junit.org/junit5/docs/current/user-guide/#junit-platform-reporting-legacy-xml. Surprising given they should be authoritative source of this file format.
* [Atlassian suggested in 2018](https://community.atlassian.com/t5/Bamboo-questions/JUnit-XML-schema/qaq-p/769965) to reference this XSD: https://github.com/windyroad/JUnit-Schema/blob/cfa434d4b8e102a8f55b8727b552a0063ee9044e/JUnit.xsd. It is a schema file so it is not as easy for humans to parse and understand the format compared to the Linux Lightning Group.

Open to suggestions